docs/iot/TOC.md

```diff
@@ -24,6 +24,7 @@
 ### [SPI](/iot/guides/peripheral-io-api-spi.md)
 ### [I2C](/iot/guides/peripheral-io-api-i2c.md)
 ### [UART](/iot/guides/peripheral-io-api-uart.md)
+## [ZigBee](/iot/guides/zigbee.md)
```

Comment:
With the rule for the TOC.md file,
would you please update this line as below?

```
## ZigBee
### [Overview](/iot/guides/zigbee.md)
```
Please check with surbhig12's comments together, which will be added.
Thanks

Reply:
Done, I also resolved @surbhig12 's comments.
Please check new patchset.